docs/stories/components/TableComplex.stories.js

```diff
@@ -7,7 +7,7 @@ import { sortBy as sort } from 'lodash';
 import { storiesOf } from '@storybook/react';
 
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faTrashAlt, faPencilAlt } from '@fortawesome/free-solid-svg-icons';
+import { faPencilAltl, faTrashAlt, faPencilAlt } from '@fortawesome/free-solid-svg-icons';
```

Comment:
The imports are correct however it's the example that is wrong can you correct it line 391?